src/widget/text-print/text-print.js

```diff
@@ -22,27 +22,25 @@ class TextPrintWidget extends Widget {
     }
 
     _addWidget() {
-        const className = 'print-text-input';
-        const printElement = document.createElement( 'div' );
-        printElement.classList.add( className, 'widget' );
+        if ( !this.widget ) {
+            const className = 'print-input-text';
+            const printElement = document.createElement( 'div' );
+            printElement.classList.add( className, 'widget' );
 
-        if ( getComputedStyle( this.element ).order !== '' ) {
-            printElement.style.order = parseInt( getComputedStyle( this.element ).order, 10 ) + 1;
-        }
-        this.element.after( printElement );
-        this.element.classList.add( 'print-only' );
+            this.element.after( printElement );
+            this.element.classList.add( 'print-hide' );
 
-        this.widget = this.element.parentElement.querySelector( `.${className}` );
-        this.widget.innerHTML = this.element.value.replace( /\n/g, '<br>' );
+            this.widget = this.element.parentElement.querySelector( `.${className}` );
+            this.widget.innerHTML = this.element.value.replace( /\n/g, '<br>' );
+        }
     }
 
     _removeWidget() {
-        this.element.classList.remove( 'print-only' );
-        this.element.parentElement.removeChild( this.widget );
-    }
-
-    update() {
```

Comment:
The update function should remain a no-op here, because the update function is called for several (non-print-related) reasons (see widgets-controller.js) and that would cause problems.